packages/strapi-plugin-i18n/services/__tests__/content-types.test.js

```diff
@@ -329,6 +329,48 @@ describe('content-types service', () => {
   });
 
   describe('copyNonLocalizedAttributes', () => {
+    test('Does not copy locale & localizations', () => {
+      const model = {
+        attributes: {
+          title: {
+            type: 'string',
+            pluginOptions: {
+              i18n: { localized: true },
+            },
+          },
+          price: {
+            type: 'integer',
+          },
+          relation: {
+            model: 'user',
+          },
+          description: {
+            type: 'string',
+          },
+          locale: {
+            type: 'string',
+          },
+          localizations: {
+            collection: 'test-model',
+          },
+        },
+      };
+
+      const input = {
+        id: 1,
+        title: 'My custom title',
+        price: 25,
+        relation: 1,
+        description: 'My super description',
```

Comment:
should we put locale and localizations there too ?

Reply:
🤦  yup